opts/config.go

```diff
@@ -0,0 +1,57 @@
+package opts
+
+import (
+	"github.com/spf13/viper"
+	"github.com/yura-under-review/port-api/repository"
+	"github.com/yura-under-review/port-api/server"
+)
+
+const (
+	defaultEnvPrefix = "APP"
+
+	defaultLogLevel = "DEBUG"
+
+	keyLogLevel   = "LOG_LEVEL"
+	keyPrettyLogs = "PRETTY_LOGS"
+
+	keyServerAddress       = "HTTP_ADDRESS"
+	keyServerRootTemplate  = "ROOT_TEMPLATE_FILE"
+	keyServerSinkBatchSize = "SINK_BATCH_SIZE"
+
+	keyPortsDomainServiceAddress = "PORTS_DOMAIN_SERVICE_ADDRESS"
+)
+
+type Config struct {
+	LogLevel   string
+	PrettyLogs bool
+
+	PortsDomainService repository.Config
+	Server             server.Config
+}
+
+func LoadConfigFromEnv() Config {
+
+	viper.AutomaticEnv()
+
+	viper.SetEnvPrefix(defaultEnvPrefix)
+
+	viper.SetDefault(keyLogLevel, defaultLogLevel)
+	viper.SetDefault(keyPrettyLogs, false)
+
+	viper.SetDefault(keyServerAddress, ":8080")
```

Comment:
I'm not a big fan of:
- using `:8080` instead of int `8080`
- setting defaults for important config values